File: node/tipo-de-cambio.js
// Tipo de cambio SUNAT del día — POST /v1/tipo-de-cambio
const token = "<tu_token>";

const cuerpo = new URLSearchParams({ fecha: "2026-07-23" });

const respuesta = await fetch("https://mblapirest.com/api/v1/tipo-de-cambio", {
  method: "POST",
  headers: {
    Authorization: `Bearer ${token}`,
    "Content-Type": "application/x-www-form-urlencoded",
  },
  body: cuerpo,
});
console.log(await respuesta.json());

File: python/tipo_de_cambio.py
"""Tipo de cambio SUNAT del día — POST /v1/tipo-de-cambio"""
import requests

token = "<tu_token>"

respuesta = requests.post(
    "https://mblapirest.com/api/v1/tipo-de-cambio",
    headers={"Authorization": f"Bearer {token}"},
    data={"fecha": "2026-07-23"},
)
print(respuesta.json())
